cpuprofilenode.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2019 Roger Chapman and the v8go contributors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package v8go
+
+// #include <stdlib.h>
+// #include "v8go.h"
+import "C"
+
+// CPUProfileNode represents a node in a call graph.
+type CPUProfileNode struct {
+	ptr C.CpuProfileNodePtr
+	iso *Isolate
```

Comment:
At what point is the `v8::CpuProfileNode`'s memory freed?  Is that when the profiler is freed or does `v8::CpuProfile::Delete` need to be called?  Either way, it seems like we need to be able to panic to avoid undefined behaviour if the `v8::CpuProfileNode` is used after its memory is freed.

Reply:
The documentation for `v8::CpuProfile::Delete` states:
```
   * Deletes the profile and removes it from CpuProfiler's list.
   * All pointers to nodes previously returned become invalid.
```

Are you suggesting that any function on CpuProfileNode in go should panic if the internal ptr to `C.CpuProfileNodePtr` is nil?